### PR TITLE
URL extraction bug fix

### PR DESCRIPTION
**Pull request by dansterenson**
Fixed the issue by adding a function to the formatter which removes the square brackets from the end of the URL.

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/47299)

Before fix:
image

After fix:
image

### CODE REVIEW

File: Packs/CommonScripts/Scripts/FormatURL/FormatURL.py [hunk 1]
```diff
@@ -102,6 +102,18 @@ def replace_protocol(url_: str) -> str:
     return url_
 
 
+def remove_brackets_from_end_of_url(url_: str) -> str:
```
Comment (yaakovi) [inline Packs/CommonScripts/Scripts/FormatURL/FormatURL.py:105]: why `url_` and not `url`?

File: Packs/CommonScripts/Scripts/FormatURL/FormatURL_test.py [hunk 2]
```diff
@@ -49,6 +49,12 @@
     # End of Sanity test, no replacement should be done.
 ]
 
+BRACKETS_URL_TO_FORMAT = [('https://test1.test-api.com/test1/test2/s.testing]',
+                           'https://test1.test-api.com/test1/test2/s.testing'),
+                          ('https://test1.test-api.com]', 'https://test1.test-api.com'),
+                          ('https://test1.test-api.com[', 'https://test1.test-api.com'),
+                          ('https://test1.test-api.com', 'https://test1.test-api.com')]
```
Comment (yaakovi) [inline Packs/CommonScripts/Scripts/FormatURL/FormatURL_test.py:56]:
```suggestion
BRACKETS_URL_TO_FORMAT = [
    ('https://test1.test-api.com/test1/test2/s.testing]', 'https://test1.test-api.com/test1/test2/s.testing'),
    ('https://test1.test-api.com/test1/test2/s]testing]', 'https://test1.test-api.com/test1/test2/s]testing'),
    ('https://test1.test-api.com/test1/test2/s]testing', 'https://test1.test-api.com/test1/test2/s]testing'),
    ('https://test1.test-api.com]', 'https://test1.test-api.com'),
    ('https://test1.test-api.com[', 'https://test1.test-api.com'),
    ('https://test1.test-api.com', 'https://test1.test-api.com'),
]
```

File: Packs/CommonScripts/ReleaseNotes/1_6_44.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### FormatURL
+- Added a function that removes square brackets from the end of the URL if there are any.
```
Comment (yaakovi) [inline Packs/CommonScripts/ReleaseNotes/1_6_44.md:4]:
```suggestion
- The script now removes a trailing square bracket from the URL if there is any.
```

File: Packs/CommonScripts/Scripts/FormatURL/FormatURL.py (hunk 1 above)
Comment (dansterenson) [reply Packs/CommonScripts/Scripts/FormatURL/FormatURL.py:105]: I followed the convention of the function above.

**Pull request merged**